<!DOCTYPE html><html><head><meta content="text/html; charset=utf-8" http-equiv="Content-Type" /><meta content="width=device-width, initial-scale=1" name="viewport" /><!--replace-start-0--><!--replace-start-5--><!--replace-start-8--><title>RNA transcription &amp; processing - My Zettelkasten</title><!--replace-end-8--><!--replace-end-5--><!--replace-end-0--><link href="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.7/dist/semantic.min.css" rel="stylesheet" /><link href="https://fonts.googleapis.com/css?family=Merriweather|Libre+Franklin|Roboto+Mono&amp;display=swap" rel="stylesheet" /><!--replace-start-1--><!--replace-start-4--><!--replace-start-7--><link href="https://raw.githubusercontent.com/srid/neuron/master/assets/neuron.svg" rel="icon" /><meta content="With the classic RNAs, i.e. messenger, ribosomal and transference, it was possible to explain the transduction of proteins. Today we know that there are many non-coding RNAs that, even though they don’t encode for a given protein, are nevertheless functional, playing an important role in the protein" name="description" /><meta content="RNA transcription &amp; processing" property="og:title" /><meta content="My Zettelkasten" property="og:site_name" /><meta content="article" property="og:type" /><script type="application/ld+json">[]</script><style type="text/css">body{background-color:#eeeeee !important;font-family:"Libre Franklin", serif !important}body .ui.container{font-family:"Libre Franklin", serif !important}body h1, h2, h3, h4, h5, h6, .ui.header, .headerFont{font-family:"Merriweather", sans-serif !important}body code, pre, tt, .monoFont{font-family:"Roboto Mono","SFMono-Regular","Menlo","Monaco","Consolas","Liberation Mono","Courier New", monospace !important}body div.z-index p.info{color:#808080}body div.z-index ul{list-style-type:square;padding-left:1.5em}body div.z-index .uplinks{margin-left:0.29999em}body .zettel-content h1#title-h1{background-color:rgba(33,133,208,0.1)}body nav.bottomPane{background-color:rgba(33,133,208,2.0e-2)}body div#footnotes{border-top-color:#2185d0}body p{line-height:150%}body img{max-width:100%}body .deemphasized{font-size:0.94999em}body .deemphasized:hover{opacity:1}body .deemphasized:not(:hover){opacity:0.69999}body .deemphasized:not(:hover) a{color:#808080 !important}body div.container.universe{padding-top:1em}body div.zettel-view ul{padding-left:1.5em;list-style-type:square}body div.zettel-view .pandoc .highlight{background-color:#ffff00}body div.zettel-view .pandoc .ui.disabled.fitted.checkbox{margin-right:0.29999em;vertical-align:middle}body div.zettel-view .zettel-content .metadata{margin-top:1em}body div.zettel-view .zettel-content .metadata div.date{text-align:center;color:#808080}body div.zettel-view .zettel-content h1{padding-top:0.2em;padding-bottom:0.2em;text-align:center}body div.zettel-view .zettel-content h2{border-bottom:solid 1px #4682b4;margin-bottom:0.5em}body div.zettel-view .zettel-content h3{margin:0px 0px 0.4em 0px}body div.zettel-view .zettel-content h4{opacity:0.8}body div.zettel-view .zettel-content div#footnotes{margin-top:4em;border-top-style:groove;border-top-width:2px;font-size:0.9em}body div.zettel-view .zettel-content div#footnotes ol > li > p:only-of-type{display:inline;margin-right:0.5em}body div.zettel-view .zettel-content aside.footnote-inline{width:30%;padding-left:15px;margin-left:15px;float:right;background-color:#d3d3d3}body div.zettel-view .zettel-content .overflows{overflow:auto}body div.zettel-view .zettel-content code{margin:auto auto auto auto;font-size:100%}body div.zettel-view .zettel-content p code, li code, ol code{padding:0.2em 0.2em 0.2em 0.2em;background-color:#f5f2f0}body div.zettel-view .zettel-content pre{overflow:auto}body div.zettel-view .zettel-content dl dt{font-weight:bold}body div.zettel-view .zettel-content blockquote{background-color:#f9f9f9;border-left:solid 10px #cccccc;margin:1.5em 0px 1.5em 0px;padding:0.5em 10px 0.5em 10px}body div.zettel-view .zettel-content.raw{background-color:#dddddd}body .ui.label.zettel-tag{color:#000000}body .ui.label.zettel-tag a{color:#000000}body nav.bottomPane ul.backlinks > li{padding-bottom:0.4em;list-style-type:disc}body nav.bottomPane ul.context-list > li{list-style-type:lower-roman}body .footer-version img{-webkit-filter:grayscale(100%);-moz-filter:grayscale(100%);-ms-filter:grayscale(100%);-o-filter:grayscale(100%);filter:grayscale(100%)}body .footer-version img:hover{-webkit-filter:grayscale(0%);-moz-filter:grayscale(0%);-ms-filter:grayscale(0%);-o-filter:grayscale(0%);filter:grayscale(0%)}body .footer-version, .footer-version a, .footer-version a:visited{color:#808080}body .footer-version a{font-weight:bold}body .footer-version{margin-top:1em !important;font-size:0.69999em}@media only screen and (max-width: 768px){body div#zettel-container{margin-left:0.4em !important;margin-right:0.4em !important}}body span.zettel-link-container span.zettel-link a{color:#2185d0;font-weight:bold;text-decoration:none}body span.zettel-link-container span.zettel-link a:hover{background-color:rgba(33,133,208,0.1)}body span.zettel-link-container span.extra{color:auto}body span.zettel-link-container.errors{border:solid 1px #ff0000}body span.zettel-link-container.errors span.zettel-link a:hover{text-decoration:none !important;cursor:not-allowed}body [data-tooltip]:after{font-size:0.69999em}body div.tag-tree div.node{font-weight:bold}body div.tag-tree div.node a.inactive{color:#555555}body .tree.flipped{-webkit-transform:rotate(180deg);-moz-transform:rotate(180deg);-ms-transform:rotate(180deg);-o-transform:rotate(180deg);transform:rotate(180deg)}body .tree{overflow:auto}body .tree ul.root{padding-top:0px;margin-top:0px}body .tree ul{position:relative;padding:1em 0px 0px 0px;white-space:nowrap;margin:0px auto 0px auto;text-align:center}body .tree ul::after{content:"";display:table;clear:both}body .tree ul:last-child{padding-bottom:0.1em}body .tree li{display:inline-block;vertical-align:top;text-align:center;list-style-type:none;position:relative;padding:1em 0.5em 0em 0.5em}body .tree li::before{content:"";position:absolute;top:0px;right:50%;border-top:solid 2px #cccccc;width:50%;height:1.19999em}body .tree li::after{content:"";position:absolute;top:0px;right:50%;border-top:solid 2px #cccccc;width:50%;height:1.19999em}body .tree li::after{right:auto;left:50%;border-left:solid 2px #cccccc}body .tree li:only-child{padding-top:0em}body .tree li:only-child::after{display:none}body .tree li:only-child::before{display:none}body .tree li:first-child::before{border-style:none;border-width:0px}body .tree li:first-child::after{border-radius:5px 0px 0px 0px}body .tree li:last-child::after{border-style:none;border-width:0px}body .tree li:last-child::before{border-right:solid 2px #cccccc;border-radius:0px 5px 0px 0px}body .tree ul ul::before{content:"";position:absolute;top:0px;left:50%;border-left:solid 2px #cccccc;width:0px;height:1.19999em}body .tree li div.forest-link{border:solid 2px #cccccc;padding:0.2em 0.29999em 0.2em 0.29999em;text-decoration:none;display:inline-block;border-radius:5px 5px 5px 5px;color:#333333;position:relative;top:2px}body .tree.flipped li div.forest-link{-webkit-transform:rotate(180deg);-moz-transform:rotate(180deg);-ms-transform:rotate(180deg);-o-transform:rotate(180deg);transform:rotate(180deg)}</style><!-- MathJax -->
<script async="" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
<!-- Prism.js -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism.min.css" rel="stylesheet" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/components/prism-core.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/plugins/autoloader/prism-autoloader.min.js"></script>
<!-- Mermaid.js -->
<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
<script>
    window.addEventListener("load", mermaid.initialize(
        {
            startOnLoad: true
        }
    ))
</script>
<!-- Custom CSS -->
<link href="./static/style.css" rel="stylesheet" />
<!--replace-end-7--><!--replace-end-4--><!--replace-end-1--></head><body><div class="ui fluid container universe"><!--replace-start-2--><!--replace-start-3--><!--replace-start-6--><nav class="flipped tree deemphasized" id="zettel-uptree" style="transform-origin: 50%"><ul class="root"><li><ul><li><div class="forest-link"><span class="zettel-link-container"><span class="zettel-link" title="2021-04-05T20:29"><a href="c5594c8f.html">Cellular and molecular biology (CBBCM)</a></span></span></div><ul><li><div class="forest-link"><span class="zettel-link-container"><span class="zettel-link" title="2021-04-05T13:33"><a href="4d2b51b4.html">First semester</a></span></span></div><ul><li><div class="forest-link"><span class="zettel-link-container"><span class="zettel-link" title="2021-04-04T23:09"><a href=".">Index</a></span></span></div></li></ul></li></ul></li></ul></li></ul></nav><div class="ui text container" id="zettel-container" style="position: relative"><div class="zettel-view"><article class="ui raised attached segment zettel-content"><div class="pandoc"><h1 id="title-h1">RNA transcription &amp; processing</h1><p>With the classic RNAs, i.e. messenger, ribosomal and transference, it was possible to explain the transduction of proteins. Today we know that there are many non-coding RNAs that, even though they don’t encode for a given protein, are nevertheless functional, playing an important role in the protein synthesis.</p><p>The difference between RNA and DNA resides in their compositional sugars, where DNA has a desoxyribose, and the the RNA a ribose. The bases are A, T, G and C for the case of DNA, but RNA has U instead of T. This two difference gives to the RNA special characteristics. DNA is very stable, in contrast with the liability of RNA. The bases are complementary (A=T, G=C, and in the case of RNA A=U).</p><p>The two strands are: <strong>sense</strong>, in the direction 5’-3’, and the <strong>antisense</strong> 3’-5’. The antisense strand is the one read for RNA synthesis.</p><h2 id="rna-synthesis">RNA synthesis</h2><p>Bases are incorporated to the 1’ carbon. To extend the strand, there is an hydroxilation between the OH-3’ and the first phosphate group of the next base, releasing one phosphate. The direction of growth is then 5’-3’.</p><h3 id="rna-polymerase">RNA polymerase</h3><p>There are many types of RNA polymerases, each of one playing an specific role in RNA synthesis. Nuclear genes are transcribed by:</p><ul><li>mRNA: RNA Pol II.</li><li>tRNA RNA Pol III.</li></ul><p>Nucleolar genes are transcribed by RNA Pol I (rRNA 5.8S, 18S and 28S).</p><p>Other nuclear genes:</p><ul><li>rRNA 5S: RNA Pol II</li><li>snRNA y snoRNA (for RNA maturation): RNA Pol I, II and III.</li></ul><p>RNA Pol II has a special characteristic, that is, it has a terminal carboxyl dominion of seven amino acids, which are repeated 52 times in mammals.</p><p>There are other RNA Pols of mitochondrion and cloroplasts, which are similar to bacterian ones. Plants has an extended family of RNA Pol IV and V.</p><table class="ui table"><thead><tr><th></th><th>Subunits</th><th>TCD</th></tr></thead><tbody><tr><td>RNA Pol I</td><td>14</td><td>-</td></tr><tr><td>RNA Pol II</td><td>12</td><td>*</td></tr><tr><td>RNA Pol III</td><td>17</td><td>-</td></tr></tbody></table><h3 id="transcription-phases">Transcription phases</h3><p>Genes contain a <strong>promoter</strong> which signals the starting point for RNA synthesis. The sequence is conserved among species. Around the promoter there occurs a coupling of <strong>general transcription factors</strong> which are specific for their RNA Pol, (TFI for RNA Pol I, etc). Later, RNA Pol is binded, forming the <strong>preinitiation complex</strong> and producing the opening of the promoter complex. <strong>Elongation</strong> is facilitated by elongation factors.</p><p>COMPLETE WITH THE TRANSCRIPTIONAL FACTOR OF THE RNA Pol II.</p><h3 id="regulation-sites-of-gene-transcription">Regulation sites of gene transcription</h3><p>The TATA box is the most common promoter. There are also proximal and far away promoters, enhancers, etc.</p><p>The starting transcriptional point is named +1. Everything down the gene is positive, and the other direction is negative.</p><p>DNA contains starting, processing and termination sequencies. The expression of genes then is regulated temporo-spatially. Not all genes are expressed at the same time. The control of gene expression occurs by the interaction of proteins, ncRNA and DNA.</p><p>The TATA box is a promoter at -25 to -35 bp and is not contiguos with the encoded part of the gene. This sequence of non coding DNA is necessary in order to maintain a correct reading frame.</p><h3 id="preinitiation-complex-formation">Preinitiation complex formation</h3><p>TATA box is read, TFIID/TBP is binded and produce a bending of the DNA strands allowing for the binding of TFIIB, which in turn accepts RNA Pol II into the complex. Later comes TFIIE and TFIIH, forming the preinitiation complex. Then is possible for the RNA Pol II to elongate the RNA strand, releasing all the the TF for the preinitiation complex, and binding elongation factors. In the initial stages of elongation, the pshosphorilation happens in the Serine-5 of the TCD of the Pol II.</p><h3 id="initiation-complex">Initiation complex</h3><p>Gene expression can be controlled by DNA, RNA and proteism:</p><ul><li>Cis elements: DNA sequences of the gene that are recognized for some molecules that modify its expression (TATA box, enhancers, silencers).</li><li>Trans elements: proteins that bind to DNA sequence (transcription factors).</li></ul><p>The beggining of elongation happens by phosphorilation of TCD Ser 5 by TFIIH (cdk7) While the RNA is growing it is immediately processed. The first processing is the edition of capping. (20-30 bp). Later, the elongation complex phosphorilates CTD Ser 2 by cdk9.</p><p>In the end, the terminal complex is formed as the termination signal is read, where the cleavage and polyadenilation happen.</p><p>Transcription can be paused (in the case of mRNA) by negative elongation factors. This can be released by other complexes with phosphorilation activity. The pausing allows for the formation of new initiation complexes.</p><h3 id="rna-maturation">RNA maturation</h3><p>The mRNA is completely transcribed, i.e. exons and introns, and other sequences. The first processing is the 5’ capping, then the cleavage at Poly(A) site, then polyadenylation, and then RNA splicing.</p><p>The <strong>capping</strong> happens after 25 to 30 bp elongation of the mRNA, guanosine capping and cap methylation happens. This requires:</p><ul><li>RNA triphosphatase</li><li>RNA guanyltransferase (RNGT)</li><li>RNA methyltransferase (RNMT)</li></ul><p>In order to do this, it is required a triphosphate group in the RNA. Only the first base has this characteristic (the rest lost the other two phsphates in order to elongate). The capping produces the adding of a methylguanosine in 5’-5’ fashion, thus forming a cap.</p><h3 id="polyadenylation">Polyadenylation</h3><p>There are specific factors that recognize the Poly(A) sequence. Factors are binded, which allow for cleavage and the addition of another polymerase, the PAP, that do not require a DNA sequence because it only adds Adenines. The great majority of mRNA has this sequences.</p><h3 id="elongation-and-termination">Elongation and termination</h3><p>After RNA processing, the mature mRNA is moved to the pore complex.</p><table class="ui table"><thead><tr><th>Molecule</th><th>Sequence size</th></tr></thead><tbody><tr><td>gene</td><td>27.000 bp (8.8 exons of 145 bp, 3365 intron bp)</td></tr><tr><td>mRNA mature</td><td>1.340 bp (5’UTR, 3’UTR, coding region, PolyA region)</td></tr><tr><td>5’UTR</td><td>700 bp</td></tr><tr><td>3’UTR</td><td>300 bp</td></tr></tbody></table><p>The 3’ end of an intron has an AG sequence, and the 5’ start of the next exon has a G. The 5’stating point of introns has a GU sequence and an A at some distance. This limits are conserved among species and genes.</p><p>The splicing happens then at the intron conserved sequences. snRNAs recongnize the exon-intron limit called U1. U2 recognizes the Adenine (with a free OH). U2 is not completely complementary, leaving the Adenine exposed, thus exposing the OH. The splicesome is formed by U4 U5 and U6 snRNA. U1 is then released and allowing for the cleavage of the intron. This processing happens while the transcription of mRNA.</p><p>Alternative splicing is possible, and, in fact, is the norm in about 95% of the genes. A precise model is not completely developed, but its is known that there are special types of long RNAs involved in the matter. It is also possible to produce alternative polyadenylation, RNA instability, etc.</p><h3 id="condensados-de-transcripción">Condensados de transcripción</h3><p>Focos con comportamiento de fusión líquida-líquida con rearreglos internos dinámicos que servirían para concentrar los factores participantes (?). Colabora en el reclutamiento de proteínas para poder entregar los factores necesarioms para lograr la transcripción.</p><h2 id="genes-trna-y-5s-rrna">Genes tRNA y 5S rRNA</h2><p>Para el gen 5S rRNA existe la caja C, ocurriendo unas 2000 veces en el cromosoma 1. En el caso de los genes de tRNA tiene la caja A y B, ocurriendo en casi todos los cromosomas excepto el Y y el 2 (creo).</p><p>They require RNA Pol III and its TFIII factors. RNA Pol II transcribe U6. tRNAs have introns at the anticodon site, which is cleaved by RNAsa (endonucleasa, a protein, not RNA).</p><h3 id="procesamiento-del-rdna">Procesamiento del rDNA</h3><p>Usan complejo de iniciación con RNA Pol I, un factor de unión upstream (UBF) y TBP. También experimentan las modificaciones.</p></div><div class="metadata"><div class="date" title="Zettel date"><time datetime="2021-04-09T08:30">2021-04-09</time></div></div></article><nav class="ui attached segment deemphasized backlinksPane" id="neuron-backlinks-pane"><h3 class="ui header"><span title="Backlinks from folgezettel parents">Uplinks</span></h3><ul class="backlinks"><li><span class="zettel-link-container folge"><span class="zettel-link" title="2021-04-05T20:29"><a href="c5594c8f.html">Cellular and molecular biology (CBBCM)</a><span data-nosnippet="" style="user-select: none; color: gray" title="Folgezettel">#</span></span></span><ul class="context-list" style="zoom: 85%;"><li class="item"><div class="pandoc"><span class="zettel-link-container folge"><span class="zettel-link" title="2021-04-09T08:30"><a href="1a5c1a0f.html">RNA transcription &amp; processing</a><span data-nosnippet="" style="user-select: none; color: gray" title="Folgezettel">#</span></span></span> — 09/04/2021</div></li></ul></li></ul></nav><nav class="ui attached segment deemphasized bottomPane" id="neuron-tags-pane"><div><span class="ui basic label zettel-tag" title="Tag">CBBCM/lecture/02</span><span class="ui basic label zettel-tag" title="Tag">cellular-biology/RNA</span><span class="ui basic label zettel-tag" title="Tag">cellular-biology/RNA-processing</span><span class="ui basic label zettel-tag" title="Tag">cellular-biology/RNA-transcription</span><span class="ui basic label zettel-tag" title="Tag">prof/Cecilia-Johnson</span></div></nav><nav class="ui bottom attached icon compact inverted menu blue" id="neuron-nav-bar"><!--replace-start-9--><a class="item" href="." title="Home"><i class="home icon"></i></a><!--replace-end-9--><a class="right item" href="impulse.html" title="Open Impulse"><i class="wave square icon"></i></a></nav></div></div><!--replace-end-6--><!--replace-end-3--><!--replace-end-2--><div class="ui center aligned container footer-version"><div class="ui tiny image"><a href="https://neuron.zettel.page"><img alt="logo" src="https://raw.githubusercontent.com/srid/neuron/master/assets/neuron.svg" title="Generated by Neuron 1.9.25.1" /></a></div></div></div></body></html>