<!DOCTYPE html><html><head><meta content="text/html; charset=utf-8" http-equiv="Content-Type" /><meta content="width=device-width, initial-scale=1" name="viewport" /><!--replace-start-0--><!--replace-start-5--><!--replace-start-8--><title>Introduction to NGS data analysis - My Zettelkasten</title><!--replace-end-8--><!--replace-end-5--><!--replace-end-0--><link href="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.7/dist/semantic.min.css" rel="stylesheet" /><link href="https://fonts.googleapis.com/css?family=Merriweather|Libre+Franklin|Roboto+Mono&amp;display=swap" rel="stylesheet" /><!--replace-start-1--><!--replace-start-4--><!--replace-start-7--><link href="https://raw.githubusercontent.com/srid/neuron/master/assets/neuron.svg" rel="icon" /><meta content="Next generation sequencing (NGS), or parallel sequencing." name="description" /><meta content="Introduction to NGS data analysis" property="og:title" /><meta content="My Zettelkasten" property="og:site_name" /><meta content="article" property="og:type" /><script type="application/ld+json">[]</script><style type="text/css">body{background-color:#eeeeee !important;font-family:"Libre Franklin", serif !important}body .ui.container{font-family:"Libre Franklin", serif !important}body h1, h2, h3, h4, h5, h6, .ui.header, .headerFont{font-family:"Merriweather", sans-serif !important}body code, pre, tt, .monoFont{font-family:"Roboto Mono","SFMono-Regular","Menlo","Monaco","Consolas","Liberation Mono","Courier New", monospace !important}body div.z-index p.info{color:#808080}body div.z-index ul{list-style-type:square;padding-left:1.5em}body div.z-index .uplinks{margin-left:0.29999em}body .zettel-content h1#title-h1{background-color:rgba(33,133,208,0.1)}body nav.bottomPane{background-color:rgba(33,133,208,2.0e-2)}body div#footnotes{border-top-color:#2185d0}body p{line-height:150%}body img{max-width:100%}body .deemphasized{font-size:0.94999em}body .deemphasized:hover{opacity:1}body .deemphasized:not(:hover){opacity:0.69999}body .deemphasized:not(:hover) a{color:#808080 !important}body div.container.universe{padding-top:1em}body div.zettel-view ul{padding-left:1.5em;list-style-type:square}body div.zettel-view .pandoc .highlight{background-color:#ffff00}body div.zettel-view .pandoc .ui.disabled.fitted.checkbox{margin-right:0.29999em;vertical-align:middle}body div.zettel-view .zettel-content .metadata{margin-top:1em}body div.zettel-view .zettel-content .metadata div.date{text-align:center;color:#808080}body div.zettel-view .zettel-content h1{padding-top:0.2em;padding-bottom:0.2em;text-align:center}body div.zettel-view .zettel-content h2{border-bottom:solid 1px #4682b4;margin-bottom:0.5em}body div.zettel-view .zettel-content h3{margin:0px 0px 0.4em 0px}body div.zettel-view .zettel-content h4{opacity:0.8}body div.zettel-view .zettel-content div#footnotes{margin-top:4em;border-top-style:groove;border-top-width:2px;font-size:0.9em}body div.zettel-view .zettel-content div#footnotes ol > li > p:only-of-type{display:inline;margin-right:0.5em}body div.zettel-view .zettel-content aside.footnote-inline{width:30%;padding-left:15px;margin-left:15px;float:right;background-color:#d3d3d3}body div.zettel-view .zettel-content .overflows{overflow:auto}body div.zettel-view .zettel-content code{margin:auto auto auto auto;font-size:100%}body div.zettel-view .zettel-content p code, li code, ol code{padding:0.2em 0.2em 0.2em 0.2em;background-color:#f5f2f0}body div.zettel-view .zettel-content pre{overflow:auto}body div.zettel-view .zettel-content dl dt{font-weight:bold}body div.zettel-view .zettel-content blockquote{background-color:#f9f9f9;border-left:solid 10px #cccccc;margin:1.5em 0px 1.5em 0px;padding:0.5em 10px 0.5em 10px}body div.zettel-view .zettel-content.raw{background-color:#dddddd}body .ui.label.zettel-tag{color:#000000}body .ui.label.zettel-tag a{color:#000000}body nav.bottomPane ul.backlinks > li{padding-bottom:0.4em;list-style-type:disc}body nav.bottomPane ul.context-list > li{list-style-type:lower-roman}body .footer-version img{-webkit-filter:grayscale(100%);-moz-filter:grayscale(100%);-ms-filter:grayscale(100%);-o-filter:grayscale(100%);filter:grayscale(100%)}body .footer-version img:hover{-webkit-filter:grayscale(0%);-moz-filter:grayscale(0%);-ms-filter:grayscale(0%);-o-filter:grayscale(0%);filter:grayscale(0%)}body .footer-version, .footer-version a, .footer-version a:visited{color:#808080}body .footer-version a{font-weight:bold}body .footer-version{margin-top:1em !important;font-size:0.69999em}@media only screen and (max-width: 768px){body div#zettel-container{margin-left:0.4em !important;margin-right:0.4em !important}}body span.zettel-link-container span.zettel-link a{color:#2185d0;font-weight:bold;text-decoration:none}body span.zettel-link-container span.zettel-link a:hover{background-color:rgba(33,133,208,0.1)}body span.zettel-link-container span.extra{color:auto}body span.zettel-link-container.errors{border:solid 1px #ff0000}body span.zettel-link-container.errors span.zettel-link a:hover{text-decoration:none !important;cursor:not-allowed}body [data-tooltip]:after{font-size:0.69999em}body div.tag-tree div.node{font-weight:bold}body div.tag-tree div.node a.inactive{color:#555555}body .tree.flipped{-webkit-transform:rotate(180deg);-moz-transform:rotate(180deg);-ms-transform:rotate(180deg);-o-transform:rotate(180deg);transform:rotate(180deg)}body .tree{overflow:auto}body .tree ul.root{padding-top:0px;margin-top:0px}body .tree ul{position:relative;padding:1em 0px 0px 0px;white-space:nowrap;margin:0px auto 0px auto;text-align:center}body .tree ul::after{content:"";display:table;clear:both}body .tree ul:last-child{padding-bottom:0.1em}body .tree li{display:inline-block;vertical-align:top;text-align:center;list-style-type:none;position:relative;padding:1em 0.5em 0em 0.5em}body .tree li::before{content:"";position:absolute;top:0px;right:50%;border-top:solid 2px #cccccc;width:50%;height:1.19999em}body .tree li::after{content:"";position:absolute;top:0px;right:50%;border-top:solid 2px #cccccc;width:50%;height:1.19999em}body .tree li::after{right:auto;left:50%;border-left:solid 2px #cccccc}body .tree li:only-child{padding-top:0em}body .tree li:only-child::after{display:none}body .tree li:only-child::before{display:none}body .tree li:first-child::before{border-style:none;border-width:0px}body .tree li:first-child::after{border-radius:5px 0px 0px 0px}body .tree li:last-child::after{border-style:none;border-width:0px}body .tree li:last-child::before{border-right:solid 2px #cccccc;border-radius:0px 5px 0px 0px}body .tree ul ul::before{content:"";position:absolute;top:0px;left:50%;border-left:solid 2px #cccccc;width:0px;height:1.19999em}body .tree li div.forest-link{border:solid 2px #cccccc;padding:0.2em 0.29999em 0.2em 0.29999em;text-decoration:none;display:inline-block;border-radius:5px 5px 5px 5px;color:#333333;position:relative;top:2px}body .tree.flipped li div.forest-link{-webkit-transform:rotate(180deg);-moz-transform:rotate(180deg);-ms-transform:rotate(180deg);-o-transform:rotate(180deg);transform:rotate(180deg)}</style><!-- MathJax -->
<script async="" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
<!-- Prism.js -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism.min.css" rel="stylesheet" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/components/prism-core.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/plugins/autoloader/prism-autoloader.min.js"></script>
<!-- Mermaid.js -->
<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
<script>
    window.addEventListener("load", mermaid.initialize(
        {
            startOnLoad: true
        }
    ))
</script>
<!-- Custom CSS -->
<link href="./static/style.css" rel="stylesheet" />
<!--replace-end-7--><!--replace-end-4--><!--replace-end-1--></head><body><div class="ui fluid container universe"><!--replace-start-2--><!--replace-start-3--><!--replace-start-6--><nav class="flipped tree deemphasized" id="zettel-uptree" style="transform-origin: 50%"><ul class="root"><li><ul><li><div class="forest-link"><span class="zettel-link-container"><span class="zettel-link" title="2021-04-04T23:43"><a href="dac1be5a.html">General genetics and genomics (CBGGG)</a></span></span></div><ul><li><div class="forest-link"><span class="zettel-link-container"><span class="zettel-link" title="2021-04-05T13:33"><a href="4d2b51b4.html">First semester</a></span></span></div><ul><li><div class="forest-link"><span class="zettel-link-container"><span class="zettel-link" title="2021-04-04T23:09"><a href=".">Index</a></span></span></div></li></ul></li></ul></li></ul></li></ul></nav><div class="ui text container" id="zettel-container" style="position: relative"><div class="zettel-view"><article class="ui raised attached segment zettel-content"><div class="pandoc"><h1 id="title-h1">Introduction to NGS data analysis</h1><p>Next generation sequencing (NGS), or parallel sequencing.</p><h2 id="secuenciación-por-síntesis">Secuenciación por síntesis</h2><p>Sequencing has improved from 1000 bp/day to more than 40 Gbp/day. In order to generate DNA molecules, we need to extend a previous one. In the 3’ there is an hydroxyl group which reacts with the 5’ of a dNTP (triphosphate nucleotide). The oxygen of the hydroxyl invades the phosphorus of the triphosphate, releasing water, pyrophosphate and a proton, thus forming the covalent bond.</p><p>The proton and the pyrophosphate can be detected for sequencing purposes. The pyrophosphate hydrolyzes in order to release energy for the next reaction.</p><h3 id="types-of-massive-sequencing">Types of massive sequencing</h3><p>There are many different massive sequencing techniques in use nowadays.</p><p><strong>Ion Torrent</strong>. Semiconductor sequencing. We detect the protons released in order to determine if a base was added (we know which nucleotides we expose the sample to).</p><p><strong>Pacific BioSciences</strong>. It detects fluorescence. Each nucleotide has a fluorophore, which is released upon DNA synthesis. The light corresponds to the type of nucleotide. The advantage is that corresponds to a real time sequencing, allowing up to 60,000 bp readings (normally up to 10,000 bp).</p><p><strong>Oxford Nanopore</strong>. A tiny pore allows the passage of a simple strand of DNA. This in turn deforms the nanopore, allowing ions to pass through. The change in potential gives us the information about the nucleotides that conforms the sequence.</p><p><strong>Illumina technology</strong>. We fragment DNA, add a known sequence (library preparation), perform a clonal amplification and finally a sequencing by synthesis.</p><p>The Library preparation (TruSeq) starts with DNA fragmentation, reparation of ends, and adding a known sequence to each end. This is done in order to amplify the DNA with a primer that recognizes the known sequencing. The sequencing is performed in two directions.</p><p>The clonal amplification happens in the flowcell where we attach the DNA because there is a complementary strand for the end to bind to. The complementary sequences bind first to one extreme and then to the other. This allows for clonal amplification in clusters. If there is a low concentration, then we have few sequencings, and even not perform at all. If the concentration is too high, the photographic step could not be able to read the data. So, it is really important to determine the correct library formation in order to avoid under or overclusterization.</p><p>During the Sequencing by synthesis we add colored nucleotides that stop the sequencing. This allows us to take pictures. Image analysis detects the fluorophores.</p><h3 id="raw-quality-control">Raw quality control</h3><p>The first quality control comes from the machine. If sequencing is correct, one color is stronger than the others. This is measured by a quality index in the PHRED scale, and is computed as <span class="math inline">\(Q\)</span>. The quality value is mapped to a numerical value depending on the vendor.</p><p><span class="math display">$$
Q = -10 \log_{10}(\pi_{e})
$$</span></p><p>Where <span class="math inline">\(\pi_e\)</span> is the error probability. A <span class="math inline">\(Q_{20} = 1/100\)</span> error, <span class="math inline">\(Q_{30} = 1/1000\)</span> error, and so on.</p><p>The sequencing capacity depends on the model, with varying amount of clustering formation, and max read length. The are new Illumina machines that uses only two fluorescent molecules (NextSeq) which have an known artifact if the extended sequences is too short (it fills with polyG). NextSeq is used for exomes. HiSeq is for big genomes. Thus, depending of the experiment is which platform I should use.</p><h2 id="ngs-pipelines">NGS pipelines</h2><p>The analysis is a little bit like the microarray one. First we collect the raw data (sequences and quality indexes), then we reduce by alignment, mapping to gene model, etc. Finally we male the specific experiment. The primary part is made by the machine. We will develop the secondary analysis.</p><p>GATK is the most common protocol for genome analysis. Tuxedo is used for RNA-seq. We will expand on GATK.</p><p>Pipeline: Preprocessing, alignment, mapping and quantification (RNA-seq) and variant calling (VCF). Quality control happens after each step.</p><p>When constructing the library we create indexes in order to mark the origin of the sample.</p><h2 id="quality-control">Quality control</h2><ol><li>Calidad de la muestra de DNA o RNA. Se revisa la degradación de la muestra por espectrofotometría o fluorimetría.</li><li>Calidad de las secuencias. Realizado por distintos programas. El mismo secuenciador entrega algunas métricas que se pueden usar, e.g. cantidad de lecturas, porcentaje de lecturas <span class="math inline">\(\geq Q_{30}\)</span>, porcentaje the clusters que pasaron (passing filter), densidad (que debe ser comparado con una tabla de valores esperados), distribución de valores de <span class="math inline">\(Q\)</span>.</li></ol><p>La calidad va a depender de la tecnología, configuración, concentración de la librería, etc.</p><p>It is also possible to analyse the <strong>KMER</strong> that refers to <strong>words of a determined K length</strong>. A well represented genome should not have enrichment of some KMERS at a given position. We can find indexes that should not be in the readings. Also, small sequences do not have enough homogeneity.</p><p>The reads are stochastic in relation to the direction of the DNA. After alignment we can determine if the read corresponded to a 5’ to 3’ direction, or the other way around.</p></div><div class="metadata"><div class="date" title="Zettel date"><time datetime="2021-04-19T08:32">2021-04-19</time></div></div></article><nav class="ui attached segment deemphasized backlinksPane" id="neuron-backlinks-pane"><h3 class="ui header"><span title="Backlinks from folgezettel parents">Uplinks</span></h3><ul class="backlinks"><li><span class="zettel-link-container folge"><span class="zettel-link" title="2021-04-04T23:43"><a href="dac1be5a.html">General genetics and genomics (CBGGG)</a><span data-nosnippet="" style="user-select: none; color: gray" title="Folgezettel">#</span></span></span><ul class="context-list" style="zoom: 85%;"><li class="item"><div class="pandoc"><span class="zettel-link-container folge"><span class="zettel-link" title="2021-04-19T08:32"><a href="1fb8a212.html">Introduction to NGS data analysis</a><span data-nosnippet="" style="user-select: none; color: gray" title="Folgezettel">#</span></span></span> — 19/04/2021</div></li></ul></li></ul></nav><nav class="ui attached segment deemphasized bottomPane" id="neuron-tags-pane"><div><span class="ui basic label zettel-tag" title="Tag">CBGGG/lecture/08</span><span class="ui basic label zettel-tag" title="Tag">genetics/NGS</span><span class="ui basic label zettel-tag" title="Tag">prof/Ricardo-Verdugo</span></div></nav><nav class="ui bottom attached icon compact inverted menu blue" id="neuron-nav-bar"><!--replace-start-9--><a class="item" href="." title="Home"><i class="home icon"></i></a><!--replace-end-9--><a class="right item" href="impulse.html" title="Open Impulse"><i class="wave square icon"></i></a></nav></div></div><!--replace-end-6--><!--replace-end-3--><!--replace-end-2--><div class="ui center aligned container footer-version"><div class="ui tiny image"><a href="https://neuron.zettel.page"><img alt="logo" src="https://raw.githubusercontent.com/srid/neuron/master/assets/neuron.svg" title="Generated by Neuron 1.9.25.1" /></a></div></div></div></body></html>